README.md

```diff
@@ -75,7 +75,7 @@ Number             | Title                                                    |
 [67](https://github.com/bitshares/bsips/pull/133) | Dynamic Market Fees | OpenLedger | Protocol | Draft
 [68](https://github.com/bitshares/bsips/pull/134) | Market Fee Based Asset | OpenLedger | Protocol | Draft
 [69](https://github.com/bitshares/bsips/issues/163) | Additional Assert Predicates | Christopher J. Sanborn | Protocol | Draft
-[70](https://github.com/bitshares/bsips/issues/170) | Lending for Margin Trading | George Harrap, Michel Santos | Protocol | Draft
+[70](hbsip-0070.md) | Peer-to-Peer Leveraged Trading | George Harrap, Michel Santos | Protocol | Draft
```

Comment:
Typo: ~~h~~bsip

Reply:
...and please add me to the authors. Thanks!

Reply:
done